desktop/core/src/desktop/lib/fs/gc/gs.py

```diff
@@ -135,7 +135,18 @@ def parent_path(path):
         parent_dir = abspath(bucket_path, key_path)
 
     return parent_dir
-  
+
+  def create_home_dir(self, home_path):
+    # When GS raz is enabled, try to create user home dir for REMOTE_STORAGE_HOME path
+    if is_raz_gs():
+      LOG.debug('Attempting to create user directory for path: %s' % home_path)
+      try:
+        self.mkdir(home_path)
+      except Exception as e:
+        LOG.exception('Failed to create user home directory for path %s with error: %s' % (home_path, str(e)))
+    else:
+      LOG.info('Create home directory is not available for GS filesystem')
```

Comment:
What is the logic? Technically, is it doable without RAZ?

Reply:
This method gets the `home_path` per user and logic is to simply create the user home directory.

Yes it is doable without RAZ to create directories also but there is no use-case scenario (with access key id and secret) for user-home directories. That concept is for RAZ only for now.